source/transfer_list.rst

```diff
@@ -164,28 +198,20 @@ The `tag_id` ranges are described in :numref:`tab_tag_id_ranges`.
    * - tag ID range
      - Description
 
-   * - 0x0 -- 0xf_ffff
-     - Standard tag id range. Any tag id in this range must first be allocated in this specification before being used. The allocation of the tag id requires the entry layout to be defined as well.
+   * - 0x0 -- 0x7f_ffff
+     - Standardized range. Any tag ID in this range must first be allocated in this specification before being used. The allocation of the tag ID requires the entry layout to be defined as well.
 
+   * - 0x80_0000 -- 0xff_efff
```

Comment:
This has 4096 ids for the non-standard range.
Previously the 0x10_0000 -- 0x10_ffff had 65536.

4096 is more than enough ids for the non-standard range, and I'm OK with that!
Just wanted to point this out in case this was not intentional.

Reply:
Right, no, that was intentional (I think it might have been a suggestion from Dan on the previous PR).

Reply:
I remember suggesting to move the non-standard range to the end but I don't remember anything about the size. Anyway, this LGTM.